package.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name": "mono",
+  "name": "consent-manager-monorepo",
   "private": true,
-  "devDependencies": {
-    "@changesets/changelog-github": "^0.2.7",
-    "@changesets/cli": "^2.10.3",
-    "@types/react": "^16.9.43",
-    "@types/react-dom": "^16.9.8",
-    "cypress": "^5.2.0",
-    "lerna": "^3.15.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
-    "tsdx": "^0.14.0",
-    "typescript": "^3.9.7"
+  "author": {
+    "name": "techboi GmbH",
+    "email": "opensource@techboi.io",
+    "url": "https://techboi.io/"
   },
-  "workspaces": [
-    "packages/*",
-    "examples/*"
+  "contributors": [
```

Comment:
This package will never be published, wondering whether it's worth maintaining a contributors section here, as we do that for individual packages in the monorepo.

Reply:
true that :)